### PR TITLE
fixed conflict

### PR DESCRIPTION
**Pull request by LeibGarber**
merged pull request

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ function App() {
             ))
           }
         </a>
+        <p>do you even <strong>G</strong>it</p>
```
Comment (nikfrank) [inline src/App.js]: strong tag -> maybe preferable to use span + CSS?

**Review by nikfrank — CHANGES_REQUESTED** [review]
let's take care of this by wed to get on the weekly

File: src/App.js (hunk 1 above)
Comment (LeibGarber) [reply src/App.js]: im on it

**Review by nikfrank — APPROVED** [review]
werd

**Pull request merged**